lib/Base/Tensor.cpp

```diff
@@ -148,6 +148,8 @@ static void dumpGenericImpl(Handle<ElemTy> handle, llvm::raw_ostream &os,
   }
 
   os << "]\n";
+
+  os.flush();
```

Comment:
looks legit, but wondering how that sneaked in :)

Reply:
I approved this internally but I was a bit reluctant to push it to OSS.  I noticed that some interaction with buck's test runner as causing llvm::outs() to not be properly flushed when running via `buck test`.  Adding this flush makes the problem go away but it also seems like a workaround for the quirks of an FB-internal tool.  Anyways, if no one minds, I'm happy to have this land.  Just giving the context :)